apps/studio/src/utils/resource.ts

```diff
@@ -35,3 +35,11 @@ export const getLinkToResource = ({
 }) => {
   return `/sites/${siteId}/${getResourceSubpath(type)}/${resourceId}`
 }
+
+export const getFolderHref = (siteId: string, folderId: string) => {
```

Comment:
shifted from `apps/studio/src/pages/sites/[siteId]/folders/[folderId]/index.tsx` to be reused